src/Totem/Features/Shared/_Layout.cshtml

```diff
@@ -45,6 +45,9 @@
     </environment>
 
     <script src="~/lib/jquery-ui-1.12.1/jquery-ui.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <!-- <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script> -->
```

Comment:
If we're no longer pulling this from the CDN, we should delete this line instead of commenting it out.

Reply:
This is related to bug #24. I approved pulling it out just to keep it cleaner and moved the link to the body of that ticket.